lib/os.js

```diff
@@ -349,9 +349,18 @@ function userInfo(options) {
   return user;
 }
 
+/**
+ * @returns {'\\.\nul' | '/dev/nul'}
+ */
+function devnull() {
+  return isWindows ? '\\\\.\\nul' : '/dev/null';
```

Comment:
Can this be moved outside of the function, similar to how `endianness()` is implemented.

Reply:
Yes. will do.